### PR TITLE
Updated Linux Academy courses

### PR DESCRIPTION
**Pull request by Chazzledazzle1**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.14
>
> For Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by Chazzledazzle1** [comment]
Forgot a bracket. Here is a corrected bullet: 
* [AIOps Essentials (Autoscaling Kubernetes with Prometheus Metrics) with Hands-On Labs (Linux Academy)]
(https://linuxacademy.com/course/using-machine-learning-to-scale-kubernetes-clusters/

File: content/en/docs/tutorials/online-training/overview.md [hunk 1]
```diff
@@ -11,10 +11,17 @@ Here are some of the sites that offer online training for Kubernetes:
 
 {{% capture body %}}
 
-* [Certified Kubernetes Administrator Preparation Course (Linux Academy)](https://linuxacademy.com/linux/training/course/name/certified-kubernetes-administrator-preparation-course)
+* [AIOps Essentials (Autoscaling Kubernetes with Prometheus Metrics) with Hands-On Labs (Linux Academy)
```
Comment (sftim) [inline content/en/docs/tutorials/online-training/overview.md]:
```suggestion
* [AIOps Essentials (Autoscaling Kubernetes with Prometheus Metrics) with Hands-On Labs (Linux Academy)]
```

(should fix the formatting)

**Comment by sftim** [comment]
This
/lgtm

**Pull request merged**